stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/channel/info/ChannelInfo.kt

```diff
@@ -122,50 +120,66 @@ public fun ChannelInfo(
         )
     }
 
-    Card(
-        modifier,
-        elevation = 8.dp,
-        shape = shape,
-        backgroundColor = ChatTheme.colors.barsBackground,
+    Box(
+        modifier = Modifier
+            .fillMaxSize()
+            .background(ChatTheme.colors.overlay.copy(alpha = 0.7f))
+            .clickable(
+                indication = null,
+                interactionSource = MutableInteractionSource(),
+                onClick = { onChannelOptionClick(Cancel) }
+            )
     ) {
-        Column(
-            modifier = Modifier.padding(top = 16.dp),
-            verticalArrangement = Arrangement.Center,
-            horizontalAlignment = Alignment.CenterHorizontally
+        Card(
+            modifier
+                .clickable(
+                    indication = null,
```

Comment:
So what's happening here is that we're making the `Box` fill the screen and adding a click listener to it.

How it works in compose is that that click listener effectively applies to the entire space of the box - the `Card` too.

But if the user taps anywhere on the card, then we shouldn't close/dismiss the. That's why we add the `clickable()` modifier on the `Card` - to intercept these clicks.

But... We hide the ripple/indication, because we don't want users to see this click. Or users shouldn't see the feedback from the card itself.

It's a bit of a weird behavior, but I'm not 100% sure if there is a better way to do it!

Reply:
Ah , must be a way to block Descendants per say  but i guess this is good for now 
since it does not add any overhead , simple solution for now